src/lobe.TestApp/Program.cs

```diff
@@ -11,33 +16,82 @@ class Program
     {
         static void Main(string[] args)
         {
-            var signatureFilePath = args[0];
-            var imageToClassify = args[1];
+            var parser = CreateParser();
 
-            ImageClassifier.Register("onnx", () => new OnnxImageClassifier());
+            parser.Invoke(args);
```

Comment:
I'm not super familiar with C#, but is it a common pattern to run everything inside of the parser?

Reply:
The command line library defines parsers and you can create handlers for commands. This is a common pattern with that library when creating cli tools